packages/fluentui/react-northstar/src/components/Pill/Pill.tsx

```diff
@@ -0,0 +1,162 @@
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as _ from 'lodash';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+import {
+  UIComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  createShorthandFactory,
+  SizeValue,
+  createShorthand,
+} from '../../utils';
+import { ComponentEventHandler, ShorthandValue, FluentComponentStaticProps } from '../../types';
+import { BoxProps } from '../Box/Box';
+
+import {
+  ComponentWithAs,
+  useAccessibility,
+  getElementType,
+  useStyles,
+  useTelemetry,
+  useFluentContext,
+  useUnhandledProps,
+} from '@fluentui/react-bindings';
+import { PillContent } from './PillContent';
+
+export interface PillProps extends UIComponentProps, ContentComponentProps<ShorthandValue<BoxProps>> {
+  /**
+   * A Pill can be sized.
+   */
+  size?: Extract<SizeValue, 'smaller' | 'small' | 'medium'>;
+
+  /**
+   * A Pill can be rounded
+   */
+  rounded?: boolean;
```

Comment:
I don't remember if it was discussed, but have we considered to make this state a default one and use "circular" to describe an another one?

- `rounded` becomes default
- default becomes `circular`

Asking because `Label` has a similar approach:

![An example of circular label](https://user-images.githubusercontent.com/14183168/111792673-c1b89880-88c4-11eb-8313-85a4f17a09af.png)

Reply:
Seems to me that we should pick either `circular` or `rounded` for both ? 🤔

Reply:
The design spec treats `rounded` as a exception being `circular` the default one and most common use of `Pill` shouldn't the default be based in the most common use?

Reply:
@miroslavstastny can you please share your thoughts?

Reply:
Discussed offline, let's use `rectangular` for the rounded rectangle. Pill will not have `circular` nor `rounded` prop.